### PR TITLE
Update browser extension doc

### PR DESCRIPTION
**Pull request by Amy-Centrality**

### CODE REVIEW

File: CENNZnet-infrastructures/CENNZnet-browser-extension.md [hunk 1]
```diff
@@ -9,10 +11,43 @@ The repo can be found [here](https://github.com/cennznet/extension)
...
+});
+```
+
+
+## Additional notes
 The CENNZnet extension is a fork of the Polkadot extension, so that it is suited for CENNZnet's dual token economy. 
 
 The API remains consistent with the Polkadot extension. Please refer to the [Polkadot extension documentation](https://polkadot.js.org/docs/extension/) for usages.
+
+Notes for users of early versions fo the CENNZnet browser extension: the extension used to require the users to manually update metatdata before using the exntension. This is no longer required.
```
Comment (KarishmaBothara) [inline CENNZnet-infrastructures/CENNZnet-browser-extension.md:53]: metatdata - typo

File: CENNZnet-infrastructures/CENNZnet-browser-extension.md [hunk 2]
```diff
@@ -9,10 +11,43 @@ The repo can be found [here](https://github.com/cennznet/extension)
...
+
+The extension's named as ``'cennznet-extension'``.
+
+## Signing transactions through the extension
+```js
+const injector = from(web3FromSource('cennznet-extension');
+const signer = injector.signer;
+
+// Create a transaction
+const transaction = api.tx.balances.transfer('5C7bLoDLAUjasvf2SPrSAZuiybFtgQ5ya5iqMgBeLzkwGRRr', 1000);
```
Comment (KarishmaBothara) [inline CENNZnet-infrastructures/CENNZnet-browser-extension.md:38]: may be lets use module which exist in our chain..
genericAsset over balance..

Comment (Amy-Centrality) [reply CENNZnet-infrastructures/CENNZnet-browser-extension.md:38]: oh true haha thanks :D

**Review by KarishmaBothara — APPROVED** [review]
lgtm

**Pull request merged**